srml/staking/src/lib.rs

```diff
@@ -64,6 +64,8 @@ const MAX_UNSTAKE_THRESHOLD: u32 = 10;
 const MAX_UNLOCKING_CHUNKS: u32 = 32;
 const MONTH_IN_SECONDS: u32 = 2_592_000;
 
+const STAKING_ID: LockIdentifier = *b"staking ";
```

Comment:
`LockIdentifier` is for differentiate the locks. If we custom a `enum_id/lock` we could build a more powerful lock.

Reply:
enum is used for composite of different lock structs, not for different locks.
We could have different ```LockIdentifer``` for same lock struct.

```LockIdentifer``` is a more straight and simple way to orgnize locks for each account.